packages/react-examples/src/react/Pickers/Pickers.doc.tsx

```diff
@@ -17,6 +19,11 @@ export const PickersPageProps: IDocPageProps = {
       code: TagPickerExampleCode,
       view: <TagPickerBasicExample />,
     },
+    {
+      title: 'Tag Picker with Inline Suggestions',
```

Comment:
nit, the Microsoft style guide says to use sentence case not title case for stuff like this...but also we're super inconsistent about that throughout the site, and this page uses title case for the other examples, so... 🤷‍♀️  (up to you whether to change it or not)

Reply:
I could sentence case this one, since the other adjacent examples could be interpreted as only title casing component names 😄